joomla/joomla-cms
# [5.1] Alert links system container [a11y]

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/42986#issuecomment-1987360423 .

### Summary of Changes
Adds the alert-link class to the string
Removes the color attribute on the link as it was overriding the specific classes for each type of alert


### Testing Instructions
You must be testing this with the latest branch!!

Go to System->Update sites and disable any update site
Go to System->Updates and check for updates

### Actual result BEFORE applying this Pull Request
### Light Mode
![image](https://github.com/joomla/joomla-cms/assets/1296369/64e8300c-2fd2-4379-8874-8004807e7097)

### Dark Mode
![image](https://github.com/joomla/joomla-cms/assets/1296369/1186cd34-1e8f-48f3-a4c8-7f0125abb57a)


### Expected result AFTER applying this Pull Request
### Light Mode
![image](https://github.com/joomla/joomla-cms/assets/1296369/190d2ccd-be23-45dc-bd09-aefc27ff8047)


### Dark Mode
![image](https://github.com/joomla/joomla-cms/assets/1296369/fef6d669-bdb5-4f01-a055-1a408699389d)



### Link to documentations
Please select:
